### ISSUE TITLE
Create intro email blast for All-MoFo

### ISSUE THREAD

**Issue by alanmoo** [mozilla/mattermoz#78]
## Introducing Mozilla Chat

tl;dr: Join us at https://chat.mozillafoundation.org/. Native apps available in "How to Join" below.

We've all used many chat clients here at Mozilla. There’s IRC, Skype, Google Chat, Slack, Telegram, WhatsApp, and Minigroups, not to mention good old-fashioned email. Each has its own strengths yet all of them combined create a weakness in the organization’s communication, with teams scattered across these services. These tools have a variety of unique features, but there seem to be a common subset of functional and ideological goals that align with those of Mozilla.

As we search for the right tool, the Engineering and Design teams have been evaluating Mattermost, an open source Slack-alternative–we can all just call it, "Chat". We’ve used it for a few months, contributed some code, and worked out the major usability bugs. Now we’d like to invite the rest of the Foundation to try it out and let us know what you think.
#### Goals of chat software
- Be open and promote inclusivity
- Reduce the need for emails
- Enable access to conversations at any time on any device (not just while online)
- Offer ability to manage conversations across projects and reduce the need for meetings.
#### Useful features of Chat
- Messaging and files
- Send messages and comments across public, private, and 1:1 channels
- Personalize notifications
- Share files and images
- Archiving and search
- Search all channels for past messages and comments
- View recent mentions of your name, username, nickname, and custom search terms
- Receive messages even if you’re offline
- Embeddable Gif support
#### How to join

Visit https://chat.mozillafoundation.org and sign up. (It’s not integrated with any existing accounts, yet.)

You can keep it in a browser tab, but you might find it more helpful to download a native app:
1. Download for your platform
   - [OSX](https://github.com/mattermost/desktop/releases/download/v1.1.0/mattermost-desktop-1.1.0-osx.tar.gz)
   - 64-bit [Windows](https://github.com/mattermost/desktop/releases/download/v1.1.0/mattermost-desktop-1.1.0-win64.zip)
   - Other Desktop releases available [here](https://github.com/mattermost/desktop/releases)
2. Install/Copy the Mattermost executable to your Applications folder.
3. If Mac OS gives you an “unsigned developer” warning while opening the app, ctrl-click on the app and click open.
4. Enter Team: `Mozilla`, URL: `https://chat.mozillafoundation.org`
5. Create an account or log in.

Mobile apps are available for iOS and Android

Once you're in, you can give feedback in the Mozilla Chat Dev channel.
#### Notifications:

Notifications can be customized on an account level and a channel level. 
- Account Level: Click the menu icon in the top left corner next to your username and check out “Account Settings”.
- Channel level: when in a channel, click the name of it at the top and look at notification preferences. You might want to disable unread notifications for certain channels unless you’re specifically mentioned.

**Comment by alanmoo** [comment]
Might obviate #14.

@davidascher you want to take a look at this?

**Comment by alanmoo** [comment]
Pending a link to the DMG, this is going to go out tomorrow unless there are any objections.

**Comment by alanmoo** [comment]
Added a line about native apps in the tl;dr and mentioned where to give feedback under "How to join"

**Comment by alanmoo** [comment]
I want to get this out the door, and a custom app binary seems to be a questionable approach, so here's some adjustments/additions I'm proposing:

#### How to join

Visit https://chat.mozillafoundation.org and sign up. (It’s not integrated with any existing accounts, yet.)

You can keep it in a browser tab, but you might find it more helpful to download a native app:
1. Download for your platform
   - [OSX](https://github.com/mattermost/desktop/releases/download/v1.1.0/mattermost-desktop-1.1.0-osx.tar.gz)
   - 64-bit [Windows](https://github.com/mattermost/desktop/releases/download/v1.1.0/mattermost-desktop-1.1.0-win64.zip)
   - Other Desktop releases available [here](https://github.com/mattermost/desktop/releases)
2. Install/Copy the Mattermost executable to your Applications folder.
3. If Mac OS gives you an “unsigned developer” warning while opening the app, ctrl-click on the app and click open.
4. Enter Team: `Mozilla`, URL: `https://chat.mozillafoundation.org`
5. Create an account or log in.

Mobile apps are available for iOS and Android

Once you're in, you can give feedback in the Mozilla Chat Dev channel.

#### Notifications:

Notifications can be customized on an account level and a channel level. 
- Account Level: Click the menu icon in the top left corner next to your username and check out “Account Settings”.
- Channel level: when in a channel, click the name of it at the top and look at notification preferences. You might want to disable unread notifications for certain channels unless you’re specifically mentioned.

**Comment by alanmoo** [comment]
@Pomax @xmatthewx @flukeout @hannahkane @simonwex Can I get a go/no-go on the edited top comment? If I don't hear back by 9AM Pacific tomorrow I'll assume we're good to go and send it out.

**Comment by Pomax** [comment]
looks good to me! one thing I'd ask for is only have an animated gif for the "we have animated gifs" part. The animation distracts from what you're trying to point out in the other examples.

**Comment by hannahkane** [comment]
Looks good to me @alanmoo.

I actually think the gifs are helpful. I'm not sure people would know how to do those things otherwise. Could live in documentation somewhere, though.

**Comment by xmatthewx** [comment]
Looks good @alanmoo. cc @kristinashu

**Comment by Pomax** [comment]
@alanmoo @hannahkane I meant the in-mattermost-chat-history animated gif links (minions/adventure time), not the animation for showing off how to do the thing being talked about.

**Comment by hannahkane** [comment]
gotcha, @Pomax. agree.

**Comment by kristinashu** [comment]
Lookin' good @alanmoo!

**Comment by alanmoo** [comment]
@pomax I see what you mean. I’ll record some new demo gifs in the morning

**Comment by alanmoo** [comment]
I've updated the gifs. We'll yes/no at stand and I'll send it out if we're good.
